docs/tutorials/dronecan_px4_flight_controller.rst

```diff
@@ -0,0 +1,449 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _dronecan_fc_tutorial:
+
+****************************************************
+DroneCAN Integration with a PX4 Flight Controller
+****************************************************
+This document provides a general procedure for integrating a Vertiq module with a PX4 flight controller using DroneCAN, also known as UAVCANv0. 
+
+Details on the DroneCAN protocol can be found on the `DroneCAN specification <https://dronecan.github.io/Specification/1._Introduction/>`_. For information on DroneCAN support on 
+Vertiq modules, refer to the :ref:`manual_dronecan` section of the Feature Reference Manual.
+
+Module Configuration and Enumeration
+=====================================
+The first step in the process of setting up a Vertiq module is configuring it through the IQ Control Center. The sections below describe these parameters and give suggestions 
+on how to set them up.
+
+Bitrate

```

Comment:
contradictory info b/w here and the dronecan section. Here says all of the modules can do it, the dronecan manual says only some can.

Reply:
Not anymore, I updated the DroneCAN section